src/locales/en.json

```
@@ -50,7 +50,7 @@
     "to": "To",
     "transactionDetails": "Transaction Details",
     "liquidityProviderFee": "Liquidity Provider Fee",
-    "priceImpact": "Price Impact",
+    "priceImpact": "Price Impact on",
```

```suggestion
    "priceImpact": "Price Impact on {token}",
```
the translation tool also supports inserting values through string templates